app/views/curation_concerns/base/_show_page_image.html.erb

```diff
@@ -9,71 +9,78 @@
       * extra_classes: optional. extra CSS calsses added to the wrapper <div> tag.
 
 %>
-<% wrapper_tag = member.representative_content_type =~ /audio/ ? 'show-page-audio-wrapper' : 'show-page-image-wrapper' %>
+<%
+  # Audio content in *this* list is only shown to staff.
+  # However, *all* users can access an audio player which is displayed above the hero image.
+  if current_ability.current_user.staff? || (member.representative_content_type =~ /audio/).nil?
+%>
```

Comment:
we do not need staff-specific logic in the template, rather we should edit the `show_thumb_member_presenters` in a presenter, so it does not include audio unless the user is staff, there.

Reply:
_show_page_image.html may not have to be changed at all in this PR? Or it can have a lot less.